content/test/integration/controller/content.spec.ts

```diff
@@ -1,41 +1,11 @@
-import { bufferToStream } from '@dcl/catalyst-storage/dist/content-item'
-import fs from 'fs'
 import fetch from 'node-fetch'
-import path from 'path'
-import { Controller } from '../../../src/controller/Controller'
 import { EnvironmentConfig } from '../../../src/Environment'
 import { makeNoopValidator } from '../../helpers/service/validations/NoOpValidator'
 import { setupTestEnvironment } from '../E2ETestEnvironment'
 
 describe('Integration - Get Content', () => {
   const getTestEnv = setupTestEnvironment()
 
-  it('calls the headContent controller when the head endpoint is requested', async () => {
-    const testFilePath = path.resolve(__dirname, '../', 'resources', 'some-text-file.txt')
-    const content = await fs.promises.readFile(testFilePath)
-    const id = 'some-id'
-
-    const getContentSpy = jest.spyOn(Controller.prototype, 'getContent')
-
-    const server = await getTestEnv()
-      .configServer()
-      .withConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION, true)
-      .andBuild()
-    await server.components.storage.storeStream(id, bufferToStream(content))
-
-    makeNoopValidator(server.components)
-
-    await server.startProgram()
-
-    const url = server.getUrl() + `/contents/${id}`
-    const res = await fetch(url, { method: 'HEAD' })
```

Comment:
This request method is needed to enable cache invalidation on cdn. Is wkc handling it ootb?

Reply:
yes, I just changed the handler to be used both for head and get